.goreleaser.yaml

```diff
@@ -63,6 +63,26 @@ brews:
     # Default is false.
     skip_upload: false
 
+# .goreleaser.yaml
+aurs:
+  -
+    name: nctl
```

Comment:
maybe put that line one line above so that it matches the `-`?

Reply:
done 👍